code/src/components/header.css

```diff
@@ -0,0 +1,43 @@
+:root {
+  --pink: #ed86be;
+  --peach: #eda686;
+  --blue: #869aed;
+  --grey: #a9a9a9;
+  --white: #ffffff;
+  --black: #000000;
+}
+
+.header {
+  display: flex;
+  justify-content: center;
+  position: relative;
+  width: 100%;
+  height: 400px;
+  background-image: url('../assets/peach_grad.jpg');
+  background-size: cover;
+  background-position: 85%;
+}
+
+.header h1 {
+  margin-top: 13%;
+  font-size: 45px;
+  padding: 2%;
+  color: var(--white);
+  text-shadow: 2px 2px 10px #ff9999;
+}
+
+/* STYLING FOR TABLET */
+@media screen and (min-width: 768px) {
+  .header h1 {
+    margin-top: 8%;
+    font-size: 55px;
+  }
+}
+
+/* STYLING FOR DESKTOP */
+@media screen and (min-width: 992px) {
+  .header h1 {
+    margin-top: 4%;
+    font-size: 55px;
+  }
+}
```

Comment:
I think the header "compresses" its text container a bit after the biggest breakpoints? The "things done" one gets hidden under the section under it when you open the site on a big screen.